### DISCUSSION TITLE
Cll of a sympy program failed

### DISCUSSION THREAD

**Discussion by walter-a111** [sympy/sympy#26826] [General]
On Sunday 16.07.2024 (US 07.16.2024) the call after the python prompt ">>>" "python polynom_test_01.py" works very well.
On Monday 17.07.2024 (US 07.16.2024) the call after the python prompt ">>>" "python polynom_test_01.py" works no more.
The Error message is: syntax error related to the filename "polynom_test_01" . Why ???

**Comment by oscarbenjamin** [comment]
GitHub issues are not for getting help with using SymPy or Python so I have converted this to a discussion.

It sounds like you are running a system shell command in the Python shell.
